settings.gradle

```diff
@@ -5,7 +5,3 @@ include 'common'
 include 'sink'
 include 'source'
 include 'system_test'
-
-include ':metrics-datadog'
-project(':metrics-datadog').projectDir = new File('./metrics/datadog')
```

Comment:
> Deprecate the `metrics/datadog` folder

Should this code be deleted entirely? If it's not tested and is unused, there's no point in keeping it.

Reply:
It was deleted as a part of the same commit. We don't utilize DataDog so there's no point in maintaining this code.

Reply:
Oh, I didn't realize that those were just the two deleted files. Thanks.

I got misled by the "deprecate" wording. Usually, it means that the functionality is still supported but its usage is discouraged.